configs/acra-keys.yaml

```diff
@@ -29,6 +29,30 @@ key_bundle_secret:
 # export private key data
 private_keys: false
 
+# try migration without writing to the output key store
```

Comment:
don't forget to re-generate this file after fixing wording

Reply:
> don't forget to re-generate this file after fixing wording

CI checks this and will not allow humans to forget.